.github/workflows/cli-tests.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           - name: linux
             image: ubuntu-20.04 # Focal Fossa
           - name: mac
-            image: macos-12
+            image: macos-13
```

Comment:
[inform] It turns out that `macos-13` gets further, because the tests don't seem to fail as fast, but it doesn't pass and just times out and gets canceled. We need to correct this or temporarily disable mac testing.